x-pack/plugins/synthetics/server/alert_rules/translations.ts

```diff
@@ -0,0 +1,165 @@
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.monitorType',
+      {
+        defaultMessage: 'Type (e.g. HTTP/TCP) of the monitor.',
+      }
+    ),
+  },
+  {
+    name: 'statusMessage',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.statusMessage',
+      {
+        defaultMessage:
+          'Status message e.g down or is below availability threshold in case of availability check or both.',
+      }
+    ),
+  },
+  {
+    name: 'latestErrorMessage',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.lastErrorMessage',
+      {
+        defaultMessage: 'Monitor latest error message',
+      }
+    ),
+  },
+  {
+    name: 'observerLocation',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.observerLocation',
+      {
+        defaultMessage: 'Observer location from which heartbeat check is performed.',
+      }
+    ),
+  },
+  {
+    name: 'observerHostname',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.observerHostname',
+      {
+        defaultMessage: 'Observer hostname from which heartbeat check is performed.',
+      }
+    ),
+  },
+  {
+    name: 'checkedAt',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.checkedAt',
+      {
+        defaultMessage: 'Timestamp at which monitor is checked.',
+      }
+    ),
+  },
+];
+
+export const commonStateTranslations = [
+  {
+    name: 'firstCheckedAt',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.firstCheckedAt',
+      {
+        defaultMessage: 'Timestamp indicating when this alert first checked',
+      }
+    ),
+  },
+  {
+    name: 'firstTriggeredAt',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.firstTriggeredAt',
+      {
+        defaultMessage: 'Timestamp indicating when the alert first triggered',
+      }
+    ),
+  },
+  {
+    name: 'currentTriggerStarted',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.currentTriggerStarted',
+      {
+        defaultMessage:
+          'Timestamp indicating when the current trigger state began, if alert is triggered',
+      }
+    ),
+  },
+  {
+    name: 'isTriggered',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.isTriggered',
+      {
+        defaultMessage: `Flag indicating if the alert is currently triggering`,
+      }
+    ),
+  },
+  {
+    name: 'lastCheckedAt',
+    description: i18n.translate(
+      'xpack.synthetics.alerts.monitorStatus.actionVariables.state.lastCheckedAt',
+      {
+        defaultMessage: `Timestamp indicating the alert's most recent check time`,
```

Comment:
```suggestion
        defaultMessage: `Timestamp indicating the alert most recent check time.`,
```
No possessive for inanimate objects